logspout/logspout.go

```diff
@@ -161,6 +173,14 @@ func httpStreamer(w http.ResponseWriter, req *http.Request, logstream chan *Log,
 	}
 }
 
+func getEtcdValueOrDefault(c *etcd.Client, key string, defaultValue string) string {
+	resp, err := c.Get(key, false, false)
+	if err != nil {
```

Comment:
We might want to check the error here, as other errors like if the server are unavailable can be globbed here,